README.md

```diff
@@ -0,0 +1,89 @@
+<div align="center">
+  <br/>
+  <a href="https://github.com/zaproxy/zap-api-nodejs" title="zaproxy">
+    <img width=186px src="assets/images/ZapNodeApi.png" alt="zap-api-nodejs logo">
+  </a>
+  <br/>
+<br/>
+<h2>OWASP ZAP Node API</h2>
```

Comment:
We should normalise how we refer to the API client implementation, currently using `Node API`, `Node Api`,  `Node.js API`, or `NodeJS`.

Reply:
That should be it.